### PR TITLE
Steam mongo prototype

### PR DESCRIPTION
**Pull request by ChrisCoffey**
This builds on top of my other outstanding pull. It implements a round-trip from the node process, to Steam, & then stores the results in the DB. 

Tomorrow I'm going to implement the event feed manager & wire it in.

### CODE REVIEW

File: middleware/streams/Runner.js [hunk 1]
```diff
@@ -6,14 +6,12 @@ const
...
     p.on(S.close, function(exitCode){
         console.log("ran process "+ i + "happily. Exited with code "+ exitCode);
     });
 };
 
-var x = _.range(10);
-_.each(x, function(n, i, ls){
-    launchLoop(ls, i);
-});
+var x = [76561198256538946];
```
Comment (ChrisCoffey) [inline middleware/streams/Runner.js]: This is just a random testing user id

File: middleware/core/models.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+var activityFeed = new mongoose.Schema({ records: [feedRecord] });
+
+var ActivityFeed = mongoose.model('ActivityFeed', activityFeed, 'activity_feed');
+var CurrentEvent = mongoose.model('CurrentEvents', activityEvent, 'current_events');
+
+exports.saveNewEvent = function(evnt, onError){
+    var ce = new CurrentEvent(evnt);
+    ce.save(onError);
+}
```
Comment (shadow7) [inline middleware/core/models.js]: unterminated statement needs a semi colon

File: middleware/core/models.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+var mongoose= require('mongoose'),
+    _ = require('underscore');
```
Comment (shadow7) [inline middleware/core/models.js]: unused import

File: middleware/core/db.js [hunk 4]
```diff
@@ -28,3 +28,5 @@ process.on(S.SIGINT, function(){
         process.exit(0); 
     });
 });
+
+require('./models')
```
Comment (shadow7) [inline middleware/core/db.js]: semicolon

File: middleware/core/models.js (hunk 2 above)
Comment (ChrisCoffey) [inline middleware/core/models.js]: I'll add the semi colon, but the automatic semicolon inference rules will handle this correctly

File: middleware/core/db.js (hunk 4 above)
Comment (ChrisCoffey) [inline middleware/core/db.js]: again, totally unnecessary but i'll add it in for sure

**Comment by ChrisCoffey** [comment]
merged & cleaned up. Back to you @shadow7

**Pull request merged**